### ISSUE TITLE
Fast crosshair glitch

### ISSUE THREAD

**Issue by andreasdc** [tr7zw/Exordium#164]
On 1.21 enabling fast crosshair option makes your crosshair invisible and when you press F5 your screen flickers with black color.

**Comment by tr7zw** [comment]
From the mod page:
Fast Crosshair will not work with ImmediatelyFast, just keep it off

**Comment by andreasdc** [comment]
> From the mod page: Fast Crosshair will not work with ImmediatelyFast, just keep it off

Is it possible to be fixed? On 1.20.2 it works without any issues 💌

**Comment by tr7zw** [comment]
No, this is a hard incompatibility that won't be worked on between IF and Exordium.

**Comment by andreasdc** [comment]
> No, this is a hard incompatibility that won't be worked on between IF and Exordium.

:( You know when 1.21.3 will be supported?

**Comment by tr7zw** [comment]
There is a branch that starts, but it doesn't render anything. Haven't had time to debug why, and then also need to update to 1.21.4. But this won't happen this week, as the other mods are not done yet/have other more important issues.

**Comment by andreasdc** [comment]
Is it possible to disable IF part? I guess it's better with Exordium.

**Comment by tr7zw** [comment]
Not that I know. Just leave it off.
